shared/actions/config/index.js

```diff
@@ -68,7 +69,7 @@ export function bootstrap (): AsyncAction {
       })
     } else {
       Promise.all(
-        [dispatch(getCurrentStatus()), dispatch(getExtendedStatus()), dispatch(getConfig())]).then(() => {
+        [dispatch(getCurrentStatus()), dispatch(getExtendedStatus()), dispatch(getConfig()), dispatch(favoriteList())]).then(() => {
```

Comment:
Probably should be in the components didmount. We need to talk about our refresh story soon